sms/v1/openapi.yaml

```diff
@@ -566,23 +590,14 @@ paths:
         - BYON Contact Service
       summary: Create a contact
       description: Creates a new contact.
+      security:
+        - ApiKeyHeader: []
       operationId: createContactUsingPOST
       requestBody:
         $ref: "#/components/requestBodies/ContactEntity"
       responses:
-        "200":
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/ContactEntity"
-            "*/*":
-              schema:
-                $ref: "#/components/schemas/ContactEntity"
         "201":
           description: Created
-        "202":
-          description: The contact was created.
```

Comment:
The errors `200` and `202` seems to be extra here. Am I right?

Reply:
yep